### DISCUSSION TITLE
Imports are not working when upgrading from 3.2.5 to 4.0.0

### DISCUSSION THREAD

**Discussion by lucasgomes-eti** [HubbellCorp/SweetBlue#8] [Q&A] [answered]
The project was working fine with version 3.2.5 but after upgrading to 4.0.0 the imports are not recognizible anymore, like the image bellow:

![Screenshot from 2022-06-17 11-10-39](https://user-images.githubusercontent.com/23144642/174315195-72f50f47-ee66-4ecc-a3e1-8e71e7a6095b.png)

Those classes are missing from the library package, in fact every class is missing on 4.0.0, when I go to the package I can see only doc files:

![Screenshot from 2022-06-17 11-13-04](https://user-images.githubusercontent.com/23144642/174315626-f2d7a6ff-123e-4c66-84b3-50eb41b5d7b6.png)

This was also the case when I tried to implement 4.0.0 as a dependency in a brand new android project, I've build the SweetBlue source code on my machine and did replace the implementation with a local .aar and everything worked fine.
I think the last artifact publish on maven in only a javaDocJar not the actual code.

**Comment by ryanhubbell** [comment, marked as answer]
Ok, I found the issue. The javadoc jar has the same name, so gradle seems to pick a jar over an aar file. Just tell gradle to pull the aar file like the below:

```implementation 'com.idevicesinc:sweetblue:4.0.0@aar'```

I'll add an issue to make sure the javadoc jar is named differently, but this should get you up and running now.

**Reply by lucasgomes-eti** [reply]
Ok, it's working, Thanks!
